onnx/onnx.proto

```diff
@@ -8,7 +8,7 @@
 
 syntax = "proto2";
 
-package onnx;
+package ONNX_NAMESPACE;
```

Comment:
oh wow - it works? is it because it gets substituted at the compilation stage? It's quite cool :)

Reply:
Yep, protoc just treats it as normal name, the generated `pb.{h,cc}` files will have `namespace ONNX_NAMESPACE`.